DQMOffline/Trigger/python/HTMonitor_cff.py

```diff
@@ -0,0 +1,226 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQMOffline.Trigger.HTMonitor_cfi import hltHTmonitoring
+
+# HLT_PFMETNoMu90_PFMHTNoMu90_IDTight
+PFMETNoMu90_PFMHTNoMu90_HTmonitoring = hltHTmonitoring.clone()
+PFMETNoMu90_PFMHTNoMu90_HTmonitoring.FolderName = cms.string('HLT/HT/PFMETNoMu90/')
+PFMETNoMu90_PFMHTNoMu90_HTmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_PFMETNoMu90_PFMHTNoMu90_IDTight_v*")
```

Comment:
@sudhaahuja I need clarification on which paths to remove, just LT_PFMETNoMu90_PFMHTNoMu90_IDTight or others as well ?